rpc/server.go

```diff
@@ -45,6 +45,7 @@ type ServerConfig struct {
 	StorageAPI modules.StorageAPI
 	NetworkAPI modules.NetworkAPI
 	CoreAPI    modules.CoreAPI
+	TxQueueAPI modules.TransactionQueueAPI
```

Comment:
full name should probably be used in config struct here too

Reply:
updated